### PR TITLE
Add Graph interface

### PR DESCRIPTION
**Pull request by gkorland**
Replace this entire comment with:
  - **Description:** Add a Graph interface
  - **Tag maintainer:** @baskaryan @hwchase17 
  - **Twitter handle:** @g_korland

### CODE REVIEW

**Comment by gkorland** [comment]
I don't think these failures are related to my Pr

File: libs/langchain/langchain/graphs/falkordb_graph.py [hunk 1]
```diff
@@ -49,12 +57,29 @@ def get_schema(self) -> str:
...
+
     def refresh_schema(self) -> None:
         """Refreshes the schema of the FalkorDB database"""
+        node_properties = self.query(node_properties_query)
+        rel_properties = self.query(rel_properties_query)
+        relationships = self.query(rel_query)
+
+        self.structured_schema = {
+            "node_props": {
+                el[0]["labels"]: el[0]["properties"] for el in node_properties
```
Comment (baskaryan) [inline libs/langchain/langchain/graphs/falkordb_graph.py:73]: seems node_properties is a list of dicts, in which case `el` is a dict?

Comment (gkorland) [reply libs/langchain/langchain/graphs/falkordb_graph.py:73]: let me first rebase and merge this PR, after the merge of this PR https://github.com/langchain-ai/langchain/pull/11122

Comment (gkorland) [reply libs/langchain/langchain/graphs/falkordb_graph.py]: @baskaryan OK, it's ready for review

**Comment by baskaryan** [comment]
@gkorland happy to take over and resolve conflicts + merge if you'd like

**Comment by gkorland** [comment]
@baskaryan thanks! just resolved the conflicts

**Pull request merged**